src/main/java/io/confluent/connect/jdbc/source/JdbcSourceTask.java

```diff
@@ -179,6 +186,10 @@ public void start(Map<String, String> properties) {
           partitions.add(Collections.singletonMap(JdbcSourceConnectorConstants.QUERY_NAME_KEY,
                                                   JdbcSourceConnectorConstants.QUERY_NAME_VALUE));
           break;
+        case STORED_PROCEDURE:
```

Comment:
The upper condition is 
`if (mode.equals(JdbcSourceTaskConfig.MODE_INCREMENTING)
        || mode.equals(JdbcSourceTaskConfig.MODE_TIMESTAMP)
        || mode.equals(JdbcSourceTaskConfig.MODE_TIMESTAMP_INCREMENTING)) {`
        
 does it mean that we will not get here in any case?